### ISSUE TITLE
Chapter 11: Freezing the Lower Layers- option 1

### ISSUE THREAD

**Issue by aramhamidi** [ageron/handson-ml#409]
Hi @ageron 
Just wondering if the following is correct:
in cells 70 and 71:
```init``` is defined 2 times. ```new_saver``` is defined but not used.

**Comment by aramhamidi** [comment]
and also in cell 69 we are telling the optimizer to train ```hidden[34]|outputs``` during training, but when defining ```reuse_vars``` we are including hidden layer 3 in ```hidden[123]``` to be reused! Wouldn't it cause a problem?

**Comment by ageron** [comment]
Thanks @aramhamidi , indeed, it seems that I forgot to delete cell 70, fortunately it does not hurt, but it does not add anything useful. I'll remove it.
However, cell 69 is intentional: I wanted to show that you can reuse layers 1, 2 and 3, and add a new layer 4 and a new output layer, then train layers 3, 4, and output. The reused layer 3 will be modified, but hopefully this will fine-tune it rather than damage it (if the learning rate is not too large).
Thanks again!

**Comment by aramhamidi** [comment]
Thank you for the explanation.
